res/modelDir/truediv/model.cfg

```diff
@@ -0,0 +1,22 @@
+[onecc]
```

Comment:
Is it an intended change? (I cannot add a comment on the filename change, so I am adding here)

[res/modelDir/README.md → res/modelDir/truediv/README.md](https://github.com/Samsung/ONE-vscode/pull/526/files#diff-07785f60e76e432eb69be5de4628f3935834e96c8e50efbdbacc25da1cd50d04)

Reply:
I created a directory, `trueDiv` and move previous files (*.md, *.tflite) into `trueDiv` directory. (The name is `trueDiv` since tflite has only one op, `trueDiv`) 

All files in `trueDiv` is a test files (currently manual testing for this PR).